gapis/api/vulkan/command_splitter.go

```diff
@@ -134,7 +136,7 @@ func (s *commandSplitter) WriteCommand(ctx context.Context, cmd api.Cmd, id api.
 	}
 	s.readMemoriesForCmd = []*api.AllocResult{}
 	s.writeMemoriesForCmd = []*api.AllocResult{}
-	out.MutateAndWrite(ctx, id, cmd)
+	out.MutateAndWrite(ctx, api.CmdNoID, cmd)
```

Comment:
I don't have the whole logic in mind, but do you really want to pass all commands with id == CmdNoID ?
(I guess so given the subsequent, related changes to remove api.CmdID arguments in other places)

Reply:
Yes, because of how transforms work, we don't want to pass WRONG IDs down the chain if we can help it. For all 'new" commands api.CmdNoID is generally correct.